bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -168,14 +171,20 @@ impl CertKeyPair {
     }
 }
 
-#[derive(Default)]
 pub struct UnsecureAcceptAllClientCertificatesHandler {}
 impl VerifyHostNameCallback for UnsecureAcceptAllClientCertificatesHandler {
     fn verify_host_name(&self, _host_name: &str) -> bool {
         true
     }
 }
 
+pub struct RejectAllClientCertificatesHandler {}
```

Comment:
```suggestion

pub struct RejectAllCertificatesHandler {}
```
This isn't specific to client certificates. And since you're changing a lot of names already, can you change UnsecureAcceptAllClientCertificatesHandler to also remove the "Client"? Probably the "Unsecure" too.